plugins/src/webrtcsink/imp.rs

```diff
@@ -510,8 +513,8 @@ impl VideoEncoder {
     fn bitrate(&self) -> i32 {
         match self.factory_name.as_str() {
             "vp8enc" | "vp9enc" => self.element.property::<i32>("target-bitrate"),
```

Comment:
were you able to test with the vaapi VP8 encoder ooc?

Reply:
No I did not test it as hardware accelerated video decoding in Browsers under Linux is always a PITA though I guess I could at least check if it works